### ISSUE TITLE
Individual page creation: The stepper for year selection doesn't work well with voice over

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-exchange/platform#740]
When creating the individual page and entering the year for relevant experiences, the stepper doesn't work well with the voice over. This might be a voice over issue. To select a year, the voice over instructed to do 2-3 keyboard hits to use the stepper and user wasn't sure what to do and ended up manually entering the year.

**Comment by greatislander** [comment]
Seems like using `<input type="numeric">` is not very helpful: https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/

We should replace any remaining instances of this.
